### PR TITLE
Admin settings

### PR DESCRIPTION
**Pull request by chandonn-zz**
Related to #1 and #3

### CODE REVIEW

**Comment by brenoalvs** [comment]
@chandonn you'll must store the select template in `/admin/partials` folder.
Also needed some code standard fixes. Please check this.

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -106,19 +106,63 @@ public function enqueue_scripts() {
...
-	public function register_monk_settings() {
+	public function monk_add_menu_page() {
+		add_menu_page(
+			'Monk Settings',
+			'Monk',
+			'manage_options',
+			'monk',
+			array( $this, 'monk_options' ),
+			'dashicons-translation',
+			3 );
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Wrong close brackets.

File: admin/class-monk-admin.php [hunk 2]
```diff
@@ -106,19 +106,63 @@ public function enqueue_scripts() {
...
+	*/
+	public function monk_options_init() {
+		register_setting( 'generalOptions', 'monk_language' );
 		add_settings_section(
-		'monk_settings_section',				 //ID
-		'Monk',									 //Title
-		array( $this, 'monk_settings_fields' ),	 //Callback function
-		'general'								 //Page into witch this options will be added
+			'monk_general_options',
+			'General options',
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Section and Fields names needs localization using `__` function

File: admin/class-monk-admin.php [hunk 3]
```diff
@@ -106,19 +106,63 @@ public function enqueue_scripts() {
...
+			array( $this, 'monk_settings_section_render'),
+			'generalOptions'
+		);
+		add_settings_field(
+			'monk_language_field',
+			'Default language',
+			array( $this, 'language_select_render' ),
+			'generalOptions',
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Don't use camelCase in slugs

File: admin/class-monk-admin.php [hunk 4]
```diff
@@ -106,19 +106,63 @@ public function enqueue_scripts() {
...
+	*/
+	public function language_select_render() {
+		require_once plugin_dir_path( __FILE__ ) . '/partials/admin-monk-select-render.php';
 	}
 
+	/**
+	 * This is the callback for the monk_geeneral_options section
+	*/
+	public function monk_settings_section_render() {
+		echo _e( 'Adjust the way you want', 'monk');
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: `_e` is a `echo` with internationalization, `echo` is not needed here.

File: admin/partials/admin-monk-options.php [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+?>
+	<div class='wrap'>
+		<h2>Monk</h2>
+		<form action="options.php" method="POST">
+			<?php
+			settings_fields( 'generalOptions' );
```
Comment (brenoalvs) [inline admin/partials/admin-monk-options.php]: Same camelCase problem here.

File: admin/partials/admin-monk-select-render.php [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	$options = get_option( 'monk_language' );
+	?>
+	<select name='monk_language[language_select]'>
```
Comment (brenoalvs) [inline admin/partials/admin-monk-select-render.php]: Don't use single quotes in HTML.
Localize with `_e` function the option tag label

**Review by brenoalvs — CHANGES_REQUESTED** [review]

File: admin/class-monk-admin.php [hunk 7]
```diff
@@ -73,7 +73,7 @@ public function enqueue_styles() {
 		 * class.
 		 */
 
-		wp_enqueue_style( $this->monk, plugin_dir_url( __FILE__ ) . 'css/monk-admin.css', array(), $this->version, 'all' );
+			wp_enqueue_style( $this->monk, plugin_dir_url( __FILE__ ) . 'css/monk-admin.css', array(), $this->version, 'all' );
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Wrong indentation here.

File: admin/partials/admin-monk-select-render.php [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	$options = get_option( 'monk_language' );
+	?>
+	<select name="monk_language[language_select]">
+		<option value="portuguese"<?php selected( $options['language_select'], "portuguese" ); ?>>Portuguese</option>
```
Comment (brenoalvs) [inline admin/partials/admin-monk-select-render.php]: In PHP functions use single quotes.
Remember in HTML = double quotes, PHP = single quotes

**Review by brenoalvs — CHANGES_REQUESTED** [review]
@chandonn some changes are not finished yet and I added a new one.

File: admin/class-monk-admin.php (hunk 7 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/partials/admin-monk-select-render.php (hunk 8 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-select-render.php]: Fixed

File: admin/class-monk-admin.php (hunk 4 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/partials/admin-monk-options.php (hunk 5 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-options.php]: Fixed

File: admin/class-monk-admin.php (hunk 3 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 2 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 1 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

**Review by brenoalvs — APPROVED** [review]

**Pull request merged**